settings-dnsrecords.lp

```diff
@@ -14,10 +14,10 @@ PageTitle = "Local DNS Settings"
 mg.include('scripts/pi-hole/lua/settings_header.lp','r')
 ?>
 <div class="row">
-    <div class="col-md-6 settings-level-1">
+    <div class="col-md-6 settings-level-0">
         <div class="box">
             <div class="box-header with-border">
-                <h3 class="box-title">Local DNS records</h3>
+                <h3 class="box-title">Local DNS records&nbsp;&nbsp;<i class="fas fa-wrench" title="This is an advanced-level setting"></i></h3>
```

Comment:
This doesn't look right.

I'm thinking about a solution, but I don't like the idea of changing the level to zero if the option is Advanced.

Reply:
I agree... We could make this a general basic level setting or move itto `DNS`.